# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296406895 18000
#      Sun Jan 30 12:01:35 2011 -0500
# Branch release-3-4-x
# Node ID f8891edcfd350c21b5fd3e1ca8f297f5903b6b2a
# Parent  c16ce72e0a22824f881e047405eb99e873341a55
txt-eng-ft.cc (ft_manager::ft_manager): omit fc_init_done from init list if fontconfig is missing

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-30  John W. Eaton  <jwe@octave.org>
+
+	* txt-eng-ft.cc (ft_manager::ft_manager): Omit fc_init_done from
+	initialization list if fontconfig is missing.
+
 2011-01-30  Pascal Dupuis  <Pascal.Dupuis@worldonline.be>
 
 	* oct-parse.yy (class stdio_stream_reader): Disallow copying.
 
 2011-01-30  John W. Eaton  <jwe@octave.org>
 
 	* graphics.cc (gnuplot_toolkit::redraw): Call __gnuplot_drawnow__,
 	not gnuplot_drawnow.
diff --git a/src/txt-eng-ft.cc b/src/txt-eng-ft.cc
--- a/src/txt-eng-ft.cc
+++ b/src/txt-eng-ft.cc
@@ -70,17 +70,20 @@ private:
 private:
 
   // No copying!
 
   ft_manager (const ft_manager&);
 
   ft_manager& operator = (const ft_manager&);
 
-  ft_manager (void) : library (), fc_init_done (false)
+  ft_manager (void) : library ()
+#if HAVE_FONTCONFIG
+    , fc_init_done (false)
+#endif
     {
       if (FT_Init_FreeType (&library))
         {
           ::error ("unable to initialize freetype library");
         }
 
 #if HAVE_FONTCONFIG
       fc_init_done = false;
